vysledek/style.css

```diff
@@ -0,0 +1,327 @@
+    text-align: center;
+    padding: 50px;
+}
+
+li {
+    list-style-type: none;
+    text-decoration: none;
+    border-bottom: solid 1px lightgray;
+}
+
+/*                  TEXT SECTION                  */
+h2 {
+    padding-left: 20px;
+}
+
+.text {
+    font-size: 1rem;
+    padding-top: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-bottom: 30px;
+    font-weight: 600;
+}
+
+.underline {
+    text-decoration: none;
+    position: relative;
+}
+
+.underline::after {
+        position: absolute;
+        content: '';
+        height: 5px;
+        bottom: -4px;
+        margin: 0 auto;
+        left: 0;
+        width: 17%;
+        background: #629edb;
+}
+
+.autor {
+    color: gray;
+    font-size: 0.9rem;
+    text-align: center;
+    margin: 0px;
+    padding-top: 3px;
+}
+
+.autor a {
+    color: gray;
+    font-size: 0.9rem;
+}
+
+/*                  GALLERY                     */
+.gallery {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    padding-top: 40px;
+}
+
+.label {
+    color: gray;
+    font-size: 0.8rem;
+    font-weight: 800;
+    text-align: center;
+    margin: 0;
+    padding: 0;
+}
+
+.description a {
+    color: gray;
+}
+
+.foto {
+    padding-bottom: 20px;
+}
+
+/*                  FOOTER                      */
+footer {
+    background-color: #f5f5f5;
+    color: gray;
+    padding: 20px;
+    font-size: 0.75rem;
+    font-weight: 700;
+
+}
+footer a {
+    color: gray;
+}
+
+
+
+/*TABLET*/
+@media (min-width: 500px) {
+    ul {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        margin: 0px;
+    }
+
+    li {
+        border: solid 0.5px lightgray;
+        border-left: none;
+        padding-left: 25px;
+        padding-right: 25px;
+        padding-top: 5px;
+        padding-bottom: 5px;
+        font-size: 1rem;
+    }
+
+    .noborder {
+        border-right: none;
+    }
+
+    .text {
+        font-size: 1.2rem;
+    }
+
+    .underline::after {
+        position: absolute;
+        content: '';
+        height: 5px;
+        bottom: -4px;
+        margin: 0 auto;
+        left: 0;
+        width: 9%;
+        background: #629edb;
+}
+
+    .gallery {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+
+    }
+
+    .foto {
+        display: flex;
+        flex-basis: 50%;
+        align-items: flex-end;
+        justify-content: center;
+        padding: 0px;
+    }
+
+    .label {
+        display: flex;
+        flex-direction: column;
+        width: 50%;
+        position: absolute;
+        background: rgba(0, 0, 0, 0.5);
+        transition: .5s ease;
+        opacity:0;
+        text-align: center;
+        padding: 20px;
+    }
+
+    .place {
+        color: white;
+        font-weight: 700;
+        font-size: 1.1rem;
+    }
+
+    .description {
+        color: white;
+    }
+
+
+    .description a {
+        color: white;
+    }
+
+    .foto:hover .label {
```

Comment:
Dobrý trik s opacitou! 👍
Ono podle zadání to mělo být vyřešené pomocí animace (řešení nasdílíme po poslední lekci, tak pak můžeš kouknout), ale pěkně sis s tím poradila 🙂